python/cuspatial/cuspatial/core/gis.py

```diff
@@ -25,10 +25,41 @@ def directed_hausdorff_distance(x, y, count):
     ----------
     {params}
 
-    returns
-    DataFrame: 'min', 'max' columns of Hausdorff distances for each polygon
+    Example
+    -------
+    Consider a pair of lines on a grid.
```

Comment:
Add a link to a definition of Hausdorff distance here. Suggestion:
```suggestion
    
           The directed Hausdorff distance from one trajectory to another is the greatest of all
           the distances from a point in the first trajectory to the closest point in the second. See
           [Wikipedia entry](https://www.google.com/search?client=safari&rls=en&q=Hausdorff+distance)
           
            Consider a pair of lines on a grid.
```

Reply:
Forgive indentation.